core/templates/dev/head/services/AssetsBackendApiService.ts

```diff
@@ -84,7 +84,7 @@ angular.module('oppia').factory('AssetsBackendApiService', [
             // not be played on Safari.
             assetBlob = new Blob([data], {type: 'audio/mpeg'});
           } else {
-            assetBlob = new Blob([data]);
+            assetBlob = new Blob([data], {type: data.type});
```

Comment:
is there any reason we have hardcoded the type for the audio assets (line 85)? (According to the above comment ther should be a type.)

Can you please check whether we can use data.type for audio assets as well? If yes, then we can remove the type check!

Reply:
Done. PTAL, thanks!

Reply:
Just to confirm, you have tested the translation in safari, right?

Reply:
No, I do not have access to Safari browser and therefore haven't tested it there. But I did check if data.type is available when data is of audio type.

Reply:
@kevintab95, Thanks for checking the value of data.type! I wanted to make sure that it's working correctly with the value of data.type as earlier it was hardcoded.

@nithusha21, Can you help us testing this functionality?

Reply:
Just to confirm... What do you want me to test on safari?

Reply:
The comment says:
`Add type for audio assets. Without this, translations can not be played on Safari.`

I guess you should check if translations work fine on Safari.

cc. @nithusha21 @Jamesjay4199